src/main/java/org/proshin/finapi/bankconnection/FpBankConnection.java

```diff
@@ -131,11 +132,15 @@ public boolean collectiveMoneyTransferSupported() {
     }
 
     @Override
-    public Iterable<Owner> owners() {
-        return new IterableJsonArray<>(
-            this.origin.getJSONArray("owners"),
-            (array, index) -> new FpOwner(array.getJSONObject(index))
-        );
+    public Optional<Iterable<Owner>> owners() {
```

Comment:
Optional of Iterable doesn't make much sense as the iterable itself can be empty. Let's maybe map `null` value to an empty Iterable?